src/lib/utils/harnesses.js

```diff
@@ -16,16 +16,19 @@ async function fetchHarnessVariants() {
   }, {});
 }
 
-const harnesses = writable([]);
 let initialized = false;
+const vehicleHarnesses = writable([]); // List of vehicle model harnesses, excluding developer and generic make harnesses
```

Comment:
Restructuring the harness data could be a single PR
- splitting the harnesses store into vehicle/generic/all
- adding the make property to the harness objects
- updating usages across repo

Reply:
The problem is I needed this refactor to be able to implement the selector properly, and it wouldn't be useful at all otherwise. 

Basically I needed to be able to get just the vehicles, not the harnesses to show in the selector.

Reply:
Moved to #46